src/Functions/EnqueueVacancyAnalyticsSummaryGenerator.cs

```diff
@@ -25,18 +27,18 @@ public EnqueueVacancyAnalyticsSummaryGenerator(ILogger<EnqueueVacancyAnalyticsSu
         }
 
         [FunctionName("EnqueueVacancyAnalyticsSummaryGenerator")]
-        public async Task Run([TimerTrigger("0 1 * * * *")] TimerInfo myTimer, ILogger log, ExecutionContext context)
+        public async Task Run([TimerTrigger("0 1 * * * *")] TimerInfo myTimer, ILogger log, ExecutionContext context,
+            [Queue(GlobalConstants.GenerateVacancyAnalyticsQueueName), StorageAccount("QueueStorage")] ICollector<string> VacancyReferenceStorageQueue)
```

Comment:
could use `IAsyncCollector` this would allow us to use `AddAsync()` and await all the tasks at the end